code/modules/media/broadcast/transmitters/broadcast.dm

```diff
@@ -50,6 +63,20 @@
 	broadcast() // Bzzt
 
 /obj/machinery/media/transmitter/broadcast/attackby(var/obj/item/W, mob/user)
+	if(panel_open && (istype(W, /obj/item/device/multitool)||istype(W, /obj/item/weapon/wirecutters)))
+		attack_hand(user)
+	if(issolder(W))
```

Comment:
Now that's just messing with me, you could have added the nanopaste thing while doin this